modules/installation-configuration-parameters.adoc

```diff
@@ -797,6 +805,14 @@ On clusters that use Kuryr, {rh-openstack} Octavia does not support availability
 |For control plane machines,  the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.
 |A list of strings, for example `["zone-1", "zone-2"]`.
 
+|`controlPlane.platform.openstack.serverGroupPolicy`
+|Server group policy to apply to the group that will contain the control plane machines in the pool. You cannot change server group policies or affiliations after creation. Supported options include `affinity`, `anti-affinity`, `soft-affinity`, and `soft-anti-affinity`. The default value is `soft-anti-affinity`. 
```

Comment:
Can we not refer to "affinity" as a supported option? That would be technically valid, but it renders the OpenStack cluster hard to upgrade.

Reply:
```suggestion
|Server group policy to apply to the group that will contain the control plane machines in the pool. You cannot change server group policies or affiliations after creation. Supported options include `anti-affinity`, `soft-affinity`, and `soft-anti-affinity`. The default value is `soft-anti-affinity`. 
```